src/main/java/pl/touk/sputnik/connector/gerrit/ReviewInputBuilder.java

```diff
@@ -1,27 +1,40 @@
 package pl.touk.sputnik.connector.gerrit;
 
-import com.google.common.base.Joiner;
-import com.google.gerrit.extensions.api.changes.ReviewInput;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
 import org.jetbrains.annotations.NotNull;
+
 import pl.touk.sputnik.review.Comment;
 import pl.touk.sputnik.review.Review;
 import pl.touk.sputnik.review.ReviewFile;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
+import com.google.common.base.Joiner;
+import com.google.gerrit.extensions.api.changes.ReviewInput;
 
+@Slf4j
+@AllArgsConstructor
 public class ReviewInputBuilder {
 
+    private final CommentFilter commentFilter;
+
     @NotNull
     public ReviewInput toReviewInput(@NotNull Review review) {
         ReviewInput reviewInput = new ReviewInput();
         reviewInput.message = Joiner.on(". ").join(review.getMessages());
-        reviewInput.labels = new HashMap<String, Short>(review.getScores());
-        reviewInput.comments = new HashMap<String, List<ReviewInput.CommentInput>>();
+        reviewInput.labels = new HashMap<>(review.getScores());
+        reviewInput.comments = new HashMap<>();
         for (ReviewFile file : review.getFiles()) {
-            List<ReviewInput.CommentInput> comments = new ArrayList<ReviewInput.CommentInput>();
+            List<ReviewInput.CommentInput> comments = new ArrayList<>();
             for (Comment comment : file.getComments()) {
+                if(commentFilter.filter(file.getReviewFilename(), comment.getLine())){
+                    log.debug("Filter comment in file {} line {}", file.getReviewFilename(), comment.getLine());
```

Comment:
This line is ambigous. Use excluded or filtered out, because message is not clear.